### PR TITLE
Update step-debugging.md

### PR DESCRIPTION
**Pull request by amitaibu**
## The Problem/Issue/Bug:

Couldn't start xdebug based on current docs

## How this PR Solves The Problem:

Adding some necessary steps and notes

### CODE REVIEW

**Comment by amitaibu** [comment]
I assume Travis is failing on something not related to this docs PR

**Comment by rfay** [comment]
markdownlint failed,
```
docs/users/step-debugging.md:46:1 MD029/ol-prefix Ordered list item prefix [Expected: 2; Actual: 1; Style: 1/2/3]
Makefile:134: recipe for target 'markdownlint' failed
make: *** [markdownlint] Error 1
```

File: docs/users/step-debugging.md [hunk 1]
```diff
@@ -40,13 +40,18 @@ PhpStorm [zero-configuration debugging](https://confluence.jetbrains.com/display
 2. Set a breakpoint.
 3. Visit a page that should stop in the breakpoint you set.
 
+Several things to note when using this recommended option:
+
+1. Make sure you have the most up to date DDEV versions.
+1. Under Run >> Edit Configurations, make sure there are no "Servers" defined. Having existing servers will override PHPstorm's ability to auto-map the incoming request to your file system.
```
Comment (amitaibu) [inline docs/users/step-debugging.md:46]:
```suggestion
2. Under Run >> Edit Configurations, make sure there are no "Servers" defined. Having existing servers will override PHPstorm's ability to auto-map the incoming request to your file system.
```

**Comment by amitaibu** [comment]
I've added also a reference to ufw - to give a sense (and possibly the solution) for cases where not request arrives PHPStorm.

**Comment by rfay** [comment]
Thanks for this @amitaibu !

**Pull request merged**